### PR TITLE
Resolve fleet cluster label saving issue

### PR DESCRIPTION
**Pull request by rak-phillip**
This resolves an issue with Fleet Cluster labels that were not saving through the UI by removing the metadata assignment and just invoking `this.normanCluster.save()`.

Two things were uncovered while investigating this one:

1. `NormanModel` doesn't contain `_labels` or `_annotation` properties. This is a concern of mine, because there was an update from not too long ago that explicitly makes use of these properties and I don't believe that to be a mistake. This leads me to believe something has changed, but I don't know what
2. Assigning metadata values to the `labels` and `annotations` properties in `NormanModel` doesn't seem to make sense before saving. The labels component invokes `value.setLabels()` on `input`; `plugins/steve/norman-class.js` simply updates `this.labels` when `setLabels()` is invoked. It looks like something changed where we might have worked with metadata, but that no longer seems to be the case.

#4310

### CODE REVIEW

**Comment by rak-phillip** [comment]
I got a little hung up on a detail in the issue that describes editing labels via Edit as YAML working as expected, but realized that the labels in the form are for the `NormanModel` and the labels in YAML describe metadata for `FleetCluster`.. two different things in this case.

File: edit/fleet.cattle.io.cluster.vue [hunk 1]
```diff
@@ -35,8 +35,8 @@ export default {
       nc.metadata = {};
     }
 
-    nc.metadata.labels = nc._labels || {};
-    nc.metadata.annotations = nc._annotations || {};
+    nc.metadata.labels = nc.labels || {};
+    nc.metadata.annotations = nc.annotations || {};
```
Comment (nwmac) [inline edit/fleet.cattle.io.cluster.vue:39]: These aren't needed any longer - lines 38 and 39 can go

**Review by nwmac — CHANGES_REQUESTED** [review]
Tests, looks good to me - 2 lines can be removed that are no longer needed

File: edit/fleet.cattle.io.cluster.vue (hunk 1 above)
Comment (rak-phillip) [reply edit/fleet.cattle.io.cluster.vue]: fixed

**Review by nwmac — APPROVED** [review]
LGTM

**Pull request merged**